CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ option(MBGL_WITH_OPENGL "Build with OpenGL renderer" ON)
 option(MBGL_WITH_EGL "Build with EGL renderer" OFF)
 option(MBGL_WITH_OSMESA "Build with OSMesa (Software) renderer" OFF)
 option(MBGL_WITH_WERROR "Make all compilation warnings errors" ON)
+option(MBGL_WITH_X11_SUPPORT "Build with X11 Support" OFF)
```

Comment:
I wonder if it should be just `MBGL_WITH_X11` like other options?

Also should these be only in `linux.cmake` as they do not seem to be relevant to other platforms?

Reply:
good point. will move